config/defaults.go

```diff
@@ -56,6 +61,18 @@ func getCacheDefaults() *CacheConfig {
 	}
 }
 
+func getCallbackDefaults(logger *zerolog.Logger) *CallbackConfig {
```

Comment:
Maybe I'm wrong, but actually this code will always generate a callback token from default admin xpub. So by default everyone will have the same token. 
IMHO it should be generated for provided admin xpub.
In the other comment I'm pointing better place to calculate it.
(And this way, we can remove the dependency to logger 🙂

Reply:
done